### DISCUSSION TITLE
OC: Failed to load configuration

### DISCUSSION THREAD

**Discussion by H-Bylewski** [Coopydood/ultimate-macOS-KVM#148] [Questions and Support]
Hi, I was happily using Macos until I tried to install some bluetooth drivers.

Now I can no longer boot to macos

Error reads: OC: Failed to load configuration

I know I should edit my config.plst (and somehow figure out whats wrong with it) but I have no idea how to find it outside macos.

I have my qcow2 files in ultimate macOS KVM directory and that's about it.

Can some one please point me in the right direction? I really don't want to start from scratch (again).

**Comment by H-Bylewski** [comment]
It turns out that this project is even better than I thought. There is a built in recovery tool XD

**Reply by eversiege** [reply]
hey, at least you can unfuck up your fuck up now!
